specification/components/schemas/responses/retrieveReferralRequest/422Response.yaml

```diff
@@ -5,7 +5,7 @@ description: |
   | Error code                         | Description                                                                                                                  |
   | ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
   | PATIENT_ERROR                      | An error occurred while retrieving the requested patient. Do not attempt again.                                              |
-  | INVALID_REQUEST_TYPE               | The `UBRN` provided exists in e-RS but does not correspond to an active request.                                             |
+  | INVALID_REQUEST_TYPE               | The `UBRN` provided exists in e-RS but does not correspond to an Advice and Guidance request.                                             |
   | NO_RELATIONSHIP                    | The user does not have a legitimate relationship with the request.                                                           |
 headers:
```

Comment:
This is in retrieveReferalRequest - should it be the retrieveAdviceAndGuidance one?

Reply:
You're correct, thanks I'll fix that